DESCRIPTION

```diff
@@ -32,7 +32,8 @@ Authors@R: c(
     person("Björn", "Sörgel", role = "aut"),
     person("Mike", "Spahr", role = "aut"),
     person("Jessica", "Strefler", role = "aut"),
-    person("Philipp", "Verpoort", role = "aut")
+    person("Philipp", "Verpoort", role = "aut"),
+    person("Robin", "Hasse", role = "aut")
```

Comment:
Aren't those files created automatically? Can we manually edit them?

Reply:
Well I think you have to manually give the name here. `buildLibrary` will then automatically update the zenodo json file and others accordingly...